elastic/apm-server
# feat: log a message if the previous fetcher failed

## Motivation/summary

If the ES fetcher fails and the apm server falls back to kibana, the ES fetcher error is lost.
Add a logger to the chained fetcher and log the error.

## Checklist



~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
~- [ ] Documentation has been updated~

For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

- run ` TestSourcemapFetcher` systemtest
- check the logs for error messages

## Related issues
